libraries/api-documenter/src/DocumentationNode.ts

```diff
@@ -3,7 +3,7 @@
 
 import {
   ApiItem,
-  ICodeLinkElement
+  IApiItemReference
 } from '@microsoft/api-extractor';
 
 export class DocumentationNode {
```

Comment:
>DocumentationNode [](start = 13, length = 17)

Get rid of this class?

Reply:
That is planned in a separate PR.  (See note on line 59)

---
In reply to: [137150240](https://github.com/Microsoft/web-build-tools/pull/357#discussion_r137150240) [](ancestors = 137150240)